client/components/public/imports/ProfileCards.jsx

```diff
@@ -10,88 +10,66 @@ export default class ProfileCards extends Component {
     this.currentProfiles = [
       {
         name: 'Millie Johnson',
-        title: 'Professor Emerita Mathematics, WWU',
-        role: 'Primary event coordinator, puzzle creator, and founder of the Great Puzzle Hunt',
+        title: 'Director, Founder, Puzzle Creator',
```

Comment:
Millie's is the only one to use `title` and no `role`. Not sure what this looks like visually (no difference? Or intentional to make her stand out?) but just wanted to mention.

Reply:
Changed hers to role with an empty title, just to be safe!